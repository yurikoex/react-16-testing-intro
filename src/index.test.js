import { App } from './index'
import React from 'react'
import { renderToString } from 'react-dom/server'

test('should render app', () => {
        expect(renderToString(<App />)).toMatchSnapshot()
})
