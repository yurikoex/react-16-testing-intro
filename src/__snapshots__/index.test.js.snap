// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render app 1`] = `"<h1 data-reactroot=\\"\\">Hello, World</h1>"`;
